www/src/pages/ar/introduction.md

```diff
@@ -0,0 +1,43 @@
+---
+title: مقدمة
+description: مقدمة إلى the T3 Stack
+layout: ../../layouts/docs.astro
+lang: ar
+dir: rtl
+---
+
+<div class="embed">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/PbjHxIuHduU" title="The best stack for your next project" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+## T3 Stack
+
+الـ T3 Stack هو Stack لتطوير تطبيقات الويب صنع بواسطة Theo ويركز على البساطة، إعادة الاستخدام، الـ Typesafety
+
+القطع الأساسية هي [**Next.js**](https://nextjs.org/) , [**TypeScript**](https://typescriptlang.org/) , [**Tailwind CSS**](https://tailwindcss.com/) وهي دائما مضافة، إذا كنت تصنع شيئا يت
+لب Backend فيمكنك اضافة [**tRPC**](https://trpc.io/) , [**Prisma**](https://prisma.io/) و [**NextAuth.js**](https://next-auth.js.org/)
+
+لعلك لاحظت أن هناك الكثير من القطع ، هذا صحيح، اَضف قطعا وازل أخرى كما تشاء فلك مُطلق الحرية
+
+## حسنا .. ما هو create-t3-app ؟ اهو قالب؟
```

Comment:
قالب

The word above isn't common, and I have asked people around me what they think of when they hear it, and non got it right.

This one is tricky because I have no suggestion, I would be lazy and add `(template)` around it, maybe leave it to future contributor to get a common definition

searching duckduckgo for it got some result, but it wasn't related to programming in some cases, so I think go on safe side https://duckduckgo.com/?q=%D9%82%D8%A7%D9%84%D8%A8&iax=images&ia=images

Reply:
```suggestion
## حسنا … ما هو create-t3-app؟ اهو قالب؟